.pre-commit-config.yaml

```diff
@@ -5,7 +5,7 @@ repos:
     hooks:
       # Run the linter.
       - id: ruff
-        args: [--fix]
+        args: [--fix, --unsafe-fixes]
```

Comment:
Why apply unsafe fixes? Or does this only affect local operations, and thus, applies them before the tests run

Reply:
I can remove that, it was just nice because I added some more ruff rules and this applied most of them automatically. 

But yet, it won't actually do anything in CI other than warn you that it tried to fix something, since the fixes won't get committed anywhere. But locally it will try to apply the unsafe-fixes if you commit with some problem.